config/paginates.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'page' => 10,
```

Comment:
```suggestion
    'ten' => 10,
```
sử dụng page như thế người khác đọc code sẽ không hiểu đưọc `config('paginates.page')` bằng bao nhiêu và mất thêm thời gian để mở file config và tìm đến `page`